symplyphysics/laws/thermodynamics/critical_point_is_isotherm_stationary_inflection_point.py

```diff
@@ -0,0 +1,93 @@
+from sympy import Eq, Derivative, solve
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    Function,
+    print_expression,
+    validate_input,
+    symbols,
+)
+from symplyphysics.laws.thermodynamics.equations_of_state import (
+    van_der_waals_state_equation as van_der_waals_law,
+)
+
+# Description
+## Critical point (in the thermodynamic sense) is such values of volume, pressure, and temperature at 
+## which only one phase exists and at the vicinity of which the physical property of the phases of the 
+## substance change dramatically. Algebraically, the critical point is the stationary inflection point
+## of the isothermal pressure-volume dependency line.
+
+# Law: (dp/dV)_T = 0, (d**2(p)/dV**2)_T = 0
+## p - pressure
+## V - volume
+## T - temperature
+## (d/dV)_T - partial derivative with respect to volume at constant temperature
+
+# Note
+## - These equations need to be solved together with the equation of state.
+
+pressure = Function("pressure", units.pressure)
+volume = Symbol("volume", units.volume)
+temperature = symbols.thermodynamics.temperature
+
+inflection_point_eqn = Eq(Derivative(pressure(volume), volume, 1), 0)
+flat_tangent_eqn = Eq(Derivative(pressure(volume), volume, 2), 0)
+
+law = inflection_point_eqn, flat_tangent_eqn
```

Comment:
This law looks quite weird.
We've been using this approach, but it was used as a condition to prove some other law. It is a good idea to extract conditions to other entities, but laws do not look suitable for this task.
I think we should add a separate folder for conditions. Also, there is no much sense in having `calculate` functions for them. Also not sure if we need tests for them.

Reply:
Yes. For example, if we separate critical point law to 3 laws, we now have a law for critical pressure. We want to derive it from van der Waals equation. We import critical point conditions and derive the critical point like you did below.